docs.feldera.com/.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
what's the impact of this? I had to upgrade to 3.12 for some stuff to work.
will this break my stuff?

Reply:
it won't